docs/secure-connections/sauce-connect/troubleshooting.md

```diff
@@ -7,11 +7,12 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-:::note --doctor flag deprecated
+Below are some tips to help troubleshoot Sauce Connect Proxy.
 
-Effective with Sauce Connect Proxy version 4.7.0, we've [deprecated the `--doctor` flag](https://changelog.saucelabs.com/en/sauce-connect-proxy-version-30JTvzO0F). Diagnostics are now automatically triggered.
-:::
+:::caution Important
+Make sure you're using the latest version of Sauce Connect Proxy, which you can download [here](/secure-connections/sauce-connect/installation/). Using older versions may cause technical issues. If you run into an error trying to launch a tunnel, this should be your first step.
```

Comment:
If you already have SC installed, will "downloading" the latest version automatically upgrade it, or is there another process for upgrading? I ended up deleting my previous install and reinstalling from scratch, so let's be clear here about what we are advising. Thanks!

Reply:
Yes you have to delete the previous install, then install the new version